### PR TITLE
Release v1.7.0

### PR DESCRIPTION
**Pull request by beatngu13**

### CODE REVIEW

File: src/test/java/de/retest/recheck/junit/jupiter/RecheckExtensionTest.java [hunk 1]
```diff
@@ -37,11 +40,11 @@
 	@BeforeEach
 	void beforeEach() {
 		context = mock( ExtensionContext.class );
-		recheckDummy = new RecheckDummy();
-		recheckDummy.recheck = mock( RecheckLifecycle.class );
-		recheckDummy.someField = mock( Runnable.class );
-		configure( recheckDummy );
-		when( context.getDisplayName() ).thenReturn( testName );
+		receheckTest = new RecheckTest();
```
Comment (diba1013) [inline src/test/java/de/retest/recheck/junit/jupiter/RecheckExtensionTest.java]: Just noticed 🙈 `recheckTest`.

Comment (beatngu13) [reply src/test/java/de/retest/recheck/junit/jupiter/RecheckExtensionTest.java]: What a typo 😄

Comment (beatngu13) [reply src/test/java/de/retest/recheck/junit/jupiter/RecheckExtensionTest.java]: Will fix this on `master`, OK?

Comment (diba1013) [reply src/test/java/de/retest/recheck/junit/jupiter/RecheckExtensionTest.java]: Sure

**Pull request merged**